CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/components/DataAccessTabPane.java

```diff
@@ -140,20 +160,39 @@ public QueryPhasePane newTab(final PluginParameters globalPrameters) {
      * 
      * @param queryPane the pane that will be added as the content of the new tab
      */
-    public void newTab(final QueryPhasePane queryPane) {
-        final Tab newTab = new Tab(String.format(TAB_TITLE, getTabPane().getTabs().size() + 1));
-        
+    public void newTab(final QueryPhasePane queryPane, final String userCaption) {
+        final Label label = new Label(userCaption);
+        final Label defaultLabel = new Label(userCaption); // `userCaption` is only needed for
+        //unit test, because the `Label` is mocked with mockConstruction containing a parameter.
```

Comment:
I would like to better understand this. Is this a limitation with Mockito or our code/implementation. What is the actual issue?

Reply:
I think it's a limitation with Mockito.MockedConstruction.  Because the Label is mocked using MockedConstruction, once with a string parameter, it didn't allow a second label to be mocked with no parameters. Perhaps there's a way to fix it but I couldn't find.

Reply:
Thanks @Auriga2 I think that this is OK, one less assertion but makes the implementation just that little bit simpler.